### DISCUSSION TITLE
CefSharp.BrowserSubprocess with type=gpu-process starts even with -disable-gpu argument

### DISCUSSION THREAD

**Discussion by zxcvqwerasdf** [cefsharp/CefSharp#4421] [General]
- **What version of the product are you using?**
    Nuget 110.0.300

- **What architecture x86 or x64?**
    x64
	
- **What version of .Net?**
    .Net 5.0
    
- **On what operating system?**
    Win10

- **Are you using `WinForms`, `WPF` or `OffScreen`?**
    OffScreen
    
- **What steps will reproduce the problem?**

```
        CefSettings cefGlobal = new CefSettings();
        CefSharpSettings.SubprocessExitIfParentProcessClosed = true;
        Cef.EnableWaitForBrowsersToClose();
        cefGlobal.UserAgent = "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/110.0.0.0 Safari/537.36";
        cefGlobal.RootCachePath = "";
        cefGlobal.CachePath = "";

        cefGlobal.UserDataPath = "";
        cefGlobal.PersistSessionCookies = false;
        cefGlobal.PersistUserPreferences = false;
        cefGlobal.SetOffScreenRenderingBestPerformanceArgs(); //Adds -disable-gpu, -disable-gpu-composting, -enable-begin-frame-scheduling
        cefGlobal.WindowlessRenderingEnabled = true;
        Cef.Initialize(cefGlobal);
```

After Cef.Initialize CefSharp.BrowserSubprocess.exe will be spawned with next arguments:
`"F:\C#\Test1\Test1\bin\x64\Debug\CefSharp.BrowserSubprocess.exe" --type=gpu-process --no-sandbox --user-agent="Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/110.0.0.0 Safari/537.36" --lang=en-US --user-data-dir="C:\Users\Hgd\AppData\Local\CEF\User Data" --cefsharpexitsub --gpu-preferences=UAAAAAAAAADgAAAYAAAAAAAAAAAAAAAAAABgAAAAAAAwAAAAAAAAAAAAAAAQAAAAAAAAAAAAAAAAAAAAAAAAAEgAAAAAAAAASAAAAAAAAAAYAAAAAgAAABAAAAAAAAAAGAAAAAAAAAAQAAAAAAAAAAAAAAAOAAAAEAAAAAAAAAABAAAADgAAAAgAAAAAAAAACAAAAAAAAAA= --use-gl=angle --use-angle=swiftshader-webgl --log-file="F:\Work\C#\Test1\Test1\bin\x64\Debug\debug.log" --mojo-platform-channel-handle=2164 --field-trial-handle=2180,i,2349857235624509341,305853515710049417,131072 --disable-features=BackForwardCache,CalculateNativeWinOcclusion,WinUseBrowserSpellChecker /prefetch:2 --host-process-id=7912`
Wasn't -disable-gpu supposed to disable the creation of that process?

**Comment by amaitland** [comment]
In future please use [Discussions](https://github.com/cefsharp/CefSharp/discussions) to ask support related  questions.

> Wasn't -disable-gpu supposed to disable the creation of that process?

No. This is a common misconception.
